tools/legacy/ScriptBackup/autogenForSwaggers/readme.md

```diff
@@ -61,4 +61,6 @@ Optional `dotNet` properties:
 |`dotNet.namespace`|a .Net namespace                |generated from a value of the `dotNet.name` property|`"Microsoft.Azure.Management.Batch"`|
 |`dotNet.ft`       |an AutoRest flattering parameter|`0`                                                 |`0`                                 |
 |`dotNet.commit`   |a specification commit id       |current commit id                                   |`undefined`                         |
-|`dotNet.autorest` |an AutoRest package             |current AutoRest                                    |`undefined`                         |
\ No newline at end of file
+|`dotNet.autorest` |an AutoRest package             |current AutoRest                                    |`undefined`                         |
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions?path=azure-sdk-for-net%2F%2Ftools%2Flegacy%2FScriptBackup%2FautogenForSwaggers%2Freadme.md)
```

Comment:
Some readme's like this one aren't all interesting from a library/sdk prospective. Do we care about having the extra noise or does it not really matter as are just collecting data anyway?

Reply:
IMO doesn't matter. A little extra noise is *shrug* when we have a clear list of where stuff can come from. Who knows, we may see something we don't expect.